jobs/views.py

```diff
@@ -1,3 +1,26 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http  import JsonResponse
+from django.views import View
+
+from jobs.models import *
+from core.utils  import signin_decorator
+
+class FollowView(View):
+    @signin_decorator
+    def post(self, request, job_id):
+        user   = request.user.id
```

Comment:
변수명을 좀 더 명확하게 작성하여 여러 메소드의 인자로 넘길 때 휴먼 에러를 줄일 수 있습니다.
이 부분이 정말 사소하지만 타입을 지정하지 않는 파이썬의 경우 특히 더 신경써주셔야 합니다!
```suggestion
        user   = request.user
```

Reply:
수정 완료하였습니다!!